promql/engine.go

```diff
@@ -350,6 +340,21 @@ func (ng *Engine) SetQueryLogger(l QueryLogger) {
 	}
 }
 
+// GetLookbackDelta returns the lookback delta of the query engine.
+func (ng *Engine) GetLookbackDelta() time.Duration {
```

Comment:
Plain getters that don't require any mutexes / special atomic code etc. are unusual in Go - instead, just exporting the struct field is more common / preferred, unless there's a special reason to abstract it behind a function.